lib/models/s3-event.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+const crypto = require('crypto');
+
+const S3Event = function (eventData, reqParams) {
+  const {reqHeaders,sourceIp} = reqParams;
+  let eventName = '';
+  const s3Object = {
+    "key": eventData.S3Item.key,
+    "sequencer": ('00' + (Math.floor(Date.now())).toString(16)).toUpperCase()
+  };
+  switch (eventData.eventType) {
+    case 'Copy':
+      eventName = 'ObjectCreated:Copy';
+      s3Object.size = eventData.S3Item.size;
+      break;
+
+    case 'Put':
+      eventName = 'ObjectCreated:Put';
+      s3Object.size = eventData.S3Item.size;
+      s3Object.eTag = eventData.S3Item.md5;
+      break;
+
+    case 'Post':
+      eventName = 'ObjectCreated:Post';
+      s3Object.size = eventData.S3Item.size;
+      s3Object.eTag = eventData.S3Item.md5;
+      break;
+
+    case 'Delete':
+      eventName = 'ObjectRemoved:Delete';
+      break;
+  }
+
+  const event = {
+    "Records": [
+      {
+        "eventVersion": "2.0",
+        "eventSource": "aws:s3",
+        "awsRegion": reqHeaders.host,
+        "eventTime": eventData.S3Item.creationDate || new Date().toISOString(),
+        "eventName": eventName,
+        "userIdentity": {
+          "principalId": 'AWS:' + randomString(21).toUpperCase(),
+        },
+        "requestParameters": {
+          "sourceIPAddress": sourceIp
+        },
+        "responseElements": {
+          "x-amz-request-id": randomString(16).toUpperCase(),
+          "x-amz-id-2": crypto.createHash('sha256').update(reqHeaders.host).digest('base64')
+        },
+        "s3": {
+          "s3SchemaVersion": "1.0",
+          "configurationId": randomString(8) + '-' + randomString(4) + '-' + randomString(4) + '-' + randomString(4) + '-' + randomString(12),
```

Comment:
This is specified as the ID of the specified bucket notification configuration ([example](https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html)), since we don't support notification configs right now can we define this as `testConfigId` instead of the arbitrary ID format you've used here?